HISTORY.rst

```diff
@@ -4,6 +4,7 @@ History
 
 0.2.0 (2019-09-30)
```

Comment:
Rather put this in a new temporary date placeholder

```rst
0.2.1 (YYYY-MM-DD)
* Modified predict
```

Also correct the Modiefied predict